VideoLocker/build.gradle

```diff
@@ -60,6 +60,7 @@ dependencies {
     // with module updates.
     compile project (':android-iconify-fontawesome')
     compile fileTree(dir: 'libs', include: '*.jar')
+    compile 'org.apache.commons:commons-lang3:3.4'
```

Comment:
Yeah, this import was accidentally readded by commit efb2bf4b3469d4eab136be2c94df36c56ad1d70d. Removed in commit 2fc30396ca0a332631e0eaf779e0f1074b907fa1.